src/Microsoft.DotNet.Darc/DarcLib/LocalGitClient.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading.Tasks;
 using Microsoft.DotNet.DarcLib.Helpers;
 using Microsoft.Extensions.Logging;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
```

Comment:
Nit: the WebApi import is back

Reply:
Jesus, how??

Thanks

Reply:
I noticed Visual Studio really like to put it in for some reason, maybe some suggestion that got auto-accepted and it brough the import? Don't know